staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper.go

```diff
@@ -75,14 +77,18 @@ var _ utilnet.Dialer = &SpdyRoundTripper{}
 
 // NewRoundTripper creates a new SpdyRoundTripper that will use
 // the specified tlsConfig.
-func NewRoundTripper(tlsConfig *tls.Config, followRedirects bool) httpstream.UpgradeRoundTripper {
-	return NewSpdyRoundTripper(tlsConfig, followRedirects)
+func NewRoundTripper(tlsConfig *tls.Config, followRedirects, validateRedirects bool) httpstream.UpgradeRoundTripper {
```

Comment:
validateRedirects doesn't seem like a helpful name since you're doing a very particular kind of validation. fixedHostname, redirectOnlyToSameHost, hostLock, ???

Reply:
how about "requireSameHostRedirects"?

Reply:
Sure.